### PR TITLE
Router complements necessary annotations in componentconfig module

### PR DESCRIPTION
**Pull request by lvchenggang**
Signed-off-by: lvchenggang lvchenggang_yewu@cmss.chinamobile.com

What type of PR is this?
/kind feature

What this PR does / why we need it:
Router complements necessary annotations in componentconfig module.
This will make it easier for users to understand.

Does this PR introduce a user-facing change?:
NONE

### CODE REVIEW

**Comment by lvchenggang** [comment]
Anybody has time to review? Could it be merged? @kadisi @kevin-wangzefeng

File: pkg/apis/componentconfig/cloudcore/v1alpha1/types.go [hunk 1]
```diff
@@ -411,10 +411,19 @@ type CloudStream struct {
 	StreamPort uint32 `json:"streamPort,omitempty"`
 }
 
+// Router indicates router module config
 type Router struct {
+	// Enable indicates whether Router is enabled,
+	// if set to false (for debugging etc.), skip checking other Router configs.
 	// default true
```
Comment (gy95) [inline pkg/apis/componentconfig/cloudcore/v1alpha1/types.go:418]: now, this is default false :)

Comment (lvchenggang) [reply pkg/apis/componentconfig/cloudcore/v1alpha1/types.go:418]: I fixed it in #3327

Comment (lvchenggang) [reply pkg/apis/componentconfig/cloudcore/v1alpha1/types.go:418]: The default value needs to be true.

**Pull request closed without merging**